kalamine/utils.py

```diff
@@ -47,15 +47,33 @@ class Layer(IntEnum):
     ALTGR = 4
     ALTGR_SHIFT = 5
 
+    @classmethod
+    def parse(cls, raw: str) -> Optional["Layer"]:
+        rawʹ = raw.casefold()
```

Comment:
this is probably a dumb question but is this trailing single quote sign a special Python stuff ?

Reply:
That is not Python specific: it a prime, and I find it very practical for closely related variables. Common practice from Haskell I guess.